circle.yml

```diff
@@ -0,0 +1,16 @@
+dependencies:
+  override:
+    - yarn
+  pre:
+    - rm -rf ~/.yarn
+    - npm install -g yarn
```

Comment:
It does. It's 21.x, and it gives the error detailed in the workarounds link above, namely:

```
The engine "node" is incompatible with this module. Expected version ">=4.3.0 <5.0.0 || >=5.10".
```

Using the workarounds in order made things work again.